### PR TITLE
fix(RHTAPREL-119): remove lingering extraData references

### PR DESCRIPTION
**Pull request by johnbieren**
Remove last remaining references to extraData as the field is now called data.

### CODE REVIEW

**Review by scoheb — DISMISSED** [review]

**Comment by johnbieren** [comment]
The testing here found a bug in collect-data (fix is https://github.com/redhat-appstudio/release-service-utils/pull/67). I am going to remove the collect-data stuff from this PR and fix it once that other PR merges

File: tasks/run-file-updates/run-file-updates.yaml
Comment (mmalina) [inline tasks/run-file-updates/run-file-updates.yaml:7]: Shouldn't this rather be treated as part of the api changes feature and thus "0.5.0"? I'd say we're not fully done with all the changes yet, so I wouldn't call this a bug. But I don't mind either way.

Comment (johnbieren) [reply tasks/run-file-updates/run-file-updates.yaml]: Changed to 0.5.0

Comment (mmalina) [reply tasks/run-file-updates/run-file-updates.yaml]: OK, thanks. We could also change the `fix:` prefix to `feat:`. But I'll leave it up to you.

**Review by mmalina — APPROVED** [review]

**Pull request merged**
